src/platform/packages/shared/kbn-discover-contextual-components/src/data_types/logs/components/summary_column/summary_column.tsx

```diff
@@ -65,8 +64,8 @@ const SummaryCell = ({
   const density = maybeNullishDensity ?? DataGridDensity.COMPACT;
   const isCompressed = density === DataGridDensity.COMPACT;
 
-  const rowHeight = maybeNullishRowHeight ?? ROWS_HEIGHT_OPTIONS.single;
-  const isSingleLine = rowHeight === ROWS_HEIGHT_OPTIONS.single || rowHeight === 1;
+  const rowHeight = maybeNullishRowHeight ?? 1;
+  const isSingleLine = rowHeight === 1;
```

Comment:
Also here (?)

Reply:
Here I would leave explicit 1, to avoid issues if someone changes `DEFAULT_ROW_COUNT` in future and it wouldn't be single line anymore. What do you think?